src/Microsoft.TemplateEngine.Utils/DefaultTemplateEngineHost.cs

```diff
@@ -1,14 +1,20 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.TemplateEngine.Abstractions;
 using Microsoft.TemplateEngine.Abstractions.PhysicalFileSystem;
 
 namespace Microsoft.TemplateEngine.Utils
 {
-    public class DefaultTemplateEngineHost : ITemplateEngineHost
+    public class DefaultTemplateEngineHost : ITemplateEngineHost, ITemplateEngineHost2
     {
+        private readonly Dictionary<string, ILogger> _diagnosticLoggers = new Dictionary<string, ILogger>();
```

Comment:
I'm really not sure if we want to keep this "named"(`Authoring` and `Install`) diagnostic...
Just always use `Logger.LogDebug`, knowing that you have to enable `Authoring` is so deep knowledge, might as well load TemplateEngine.sln into VS and debug it...

Reply:
I would agree, for the current state of logging implementation. 
In the future we might benefit from different categories of logging, by setting them on different level:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0#create-logs-1